README.md

```diff
@@ -1,18 +1,56 @@
 # DER-scheduling
+IEC-scheduling (or DER-scheduling) is a specific form of scheduling with a lot of properties that should not be confused with any [generic scheduling](https://en.wikipedia.org/wiki/Schedule) such also as in IT.
 
 ## Introduction
+This project aims to create a production grade open-source scheduling software stack, enabling consumers, manufacturers and automations suppliers to implement IEC Scheduling according to IEC 61850-7-4-2010 Annex K. Formaly known as TR IEC 61850-90-10. The main use-case is for controling Distributed Energy Resources (DER), but not limited to that. 
 
-This project aims to create an Open source scheduling stack for distributed energy resources (DER) control according to IEC 61850-90-10.
+## Targeted applicability
+- DER = Distributed Energy Resources (e.g. PV solar, Windpower, Battery infeed into distribution grid)
+- EMS = (Plant / Industrial) Energy Managment Systems
+- HEMS = Home Energy Management Systems
+- Specific solutions such as the [FNN-steurbox](https://www.vde.com/de/fnn/arbeitsgebiete/digitalisierung-metering/lastenhefte/steuerbox) in Germany
 
-The main functionalities will be to control a DERs regarding:
+## Why IEC scheduling?
+IEC scheduling solves two structural problems in controling complex energy systems where more than one actor and more than one use-case is involved (*...and that is nearly almost the case*).
+
+**Problem 1**: There are many controls from remote, and in the local device itself. How do I get myself into control what is happening with all those loose controls? Otherwise, I won't understand the behavior and lose control.
+
+![Several actors with several capacity envelopes](./images/readme_totalenvelopes.png)
+
+
+**Answer**: IEC scheduling completely changes the concept of controlling, compared to what is known till sofar in the IEC landscape for energy automation. The main principle of it is to place ALL controls into parallell schedules and where a policy defines the priority of those schedules. There are NO direct controls. All controls, even locally generated are placed in (separate) schedules. Included in defining this policy is to give priority to schedules dependent on situational circumstances, such as:
```

Comment:
The last sentence of the answer starting with `Included in defining this policy ...` is not clear to me. I think the word including leads to the thinking policies are just another schedule.